src/panels/lovelace/badges/hui-entity-badge.ts

```diff
@@ -219,12 +219,12 @@ export class HuiEntityBadge extends LitElement implements LovelaceBadge {
         align-items: center;
         justify-content: center;
         gap: 8px;
-        height: 36px;
-        min-width: 36px;
+        height: var(--ha-badge-size, 36px);
+        min-width: var(--ha-badge-size, 36px);
         padding: 0px 8px;
         box-sizing: border-box;
         width: auto;
-        border-radius: 18px;
+        border-radius: calc(var(--ha-badge-size, 36px) / 2);
```

Comment:
This is awesome, personally I wanted to increase the badge size while also reducing the radius to make them look more square.
Could you also add a separate variable that would control the border radius and fallback to this calculated one based on  `--ha-badge-size` if not specified? Maybe like this?
```suggestion
        border-radius: var(--ha-badge-border-radius, calc(var(--ha-badge-size, 36px) / 2));
```